dev.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+import 'dart:io';
+
+void main(List<String> arguments) async {
+  if (arguments == null) {
+    throw ArgumentError('dart dev.dart <proj-name-from-project_config.json>');
+  }
+
+  var baseConfigFile = File('config/base_config.json');
+  var projConfigFile = File('config/project_config.json');
+  if (!await baseConfigFile.exists() || !await projConfigFile.exists()) {
+    throw ArgumentError('Config files not present');
```

Comment:
Aim to make this error message a bit more comprehensive by testing the existence of both files separately (if the user gets this error, they wouldn't know which of the files is missing) and by listing the full path of the file so that the user would know where it should be added.

nit: "not present" as a phrase is a bit ambiguous - it could be interpreted as "not present in the argument list". Consider using something like "Could not find config file: <path>" or "Config file doesn't exist: <path>"

Reply:
That makes sense, separated out both of them to give unambiguous error messages. Fixed